### DISCUSSION TITLE
Question about firmware

### DISCUSSION THREAD

**Discussion by ghost** [community/community#129879] [Programming Help]
### Body

I want to use C++ and a firmware library to make a program that is malware that can overwrite the CMOS with bad settings, shut down the computer, cause post fails and overwrite the flash drive to destroy my VM and destroy my older brother’s computer. It will spread through IPs by sending a request with the machine code to my brother’s computer. What parts could use Java to add more portability?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by floris-xlx** [comment]
i would advice you to not discuss this on the public internet as the internet footprint on this does not look good:)

i also have no idea why you would want to use Java, stick to cpp or another low level language
